httpserver/http_connection.cpp

```diff
@@ -208,7 +208,7 @@ void run(boost::asio::io_context& ioc, std::string& ip, uint16_t port,
 
     tcp::acceptor acceptor{ioc, {address, port}};
 
-    ssl::context ssl_ctx{ssl::context::sslv23};
```

Comment:
Let's use `tlsv13`!

Reply:
We can't unless we change our min. requirements for boost to 1.69 :(

Reply:
Looks to me that being able to use TLS1.3 is a compelling enough reason to update boost.

Reply:
At least we should make it a TODO to update to TLS 1.3. It is supposed to be much harder to misuse.